<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>691</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>jjTerm</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/terminal16.png</normaloff>:/terminal16.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>165</width>
           <height>335</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>161</width>
            <height>331</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le0">
               <property name="font">
                <font>
                 <family>Fixedsys Excelsior 3.01-L2</family>
                 <pointsize>12</pointsize>
                 <stylestrategy>NoAntialias</stylestrategy>
                 <kerning>false</kerning>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le1">
               <property name="font">
                <font>
                 <family>Fixedsys Excelsior 3.01-L2</family>
                 <pointsize>12</pointsize>
                 <stylestrategy>NoAntialias</stylestrategy>
                 <kerning>false</kerning>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le2">
               <property name="font">
                <font>
                 <family>Fixedsys Excelsior 3.01-L2</family>
                 <pointsize>12</pointsize>
                 <stylestrategy>NoAntialias</stylestrategy>
                 <kerning>false</kerning>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>3</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le3">
               <property name="font">
                <font>
                 <family>Fixedsys Excelsior 3.01-L2</family>
                 <pointsize>12</pointsize>
                 <stylestrategy>NoAntialias</stylestrategy>
                 <kerning>false</kerning>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>4</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le4">
               <property name="font">
                <font>
                 <family>Fixedsys Excelsior 3.01-L2</family>
                 <pointsize>12</pointsize>
                 <stylestrategy>NoAntialias</stylestrategy>
                 <kerning>false</kerning>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>5</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le5">
               <property name="font">
                <font>
                 <family>Fixedsys Excelsior 3.01-L2</family>
                 <pointsize>12</pointsize>
                 <stylestrategy>NoAntialias</stylestrategy>
                 <kerning>false</kerning>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>6</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le6">
               <property name="font">
                <font>
                 <family>Fixedsys Excelsior 3.01-L2</family>
                 <pointsize>12</pointsize>
                 <stylestrategy>NoAntialias</stylestrategy>
                 <kerning>false</kerning>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>7</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le7">
               <property name="font">
                <font>
                 <family>Fixedsys Excelsior 3.01-L2</family>
                 <pointsize>12</pointsize>
                 <stylestrategy>NoAntialias</stylestrategy>
                 <kerning>false</kerning>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>8</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le8">
               <property name="font">
                <font>
                 <family>Fixedsys Excelsior 3.01-L2</family>
                 <pointsize>12</pointsize>
                 <stylestrategy>NoAntialias</stylestrategy>
                 <kerning>false</kerning>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>9</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le9">
               <property name="font">
                <font>
                 <family>Fixedsys Excelsior 3.01-L2</family>
                 <pointsize>12</pointsize>
                 <stylestrategy>NoAntialias</stylestrategy>
                 <kerning>false</kerning>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>276</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="btClose">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>280</y>
            <width>61</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>Close</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="btClear">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>308</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="btScroll">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>280</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Stop scroll</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="btOpen">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>195</y>
            <width>151</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="btGraph1">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>240</y>
            <width>41</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="btGraph2">
          <property name="geometry">
           <rect>
            <x>66</x>
            <y>240</y>
            <width>41</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>1</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="btGraph3">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>240</y>
            <width>41</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>2</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QFrame" name="frG0">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>270</y>
            <width>41</width>
            <height>8</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 0, 0);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QFrame" name="frG1">
          <property name="geometry">
           <rect>
            <x>66</x>
            <y>270</y>
            <width>41</width>
            <height>8</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 0, 0);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QFrame" name="frG2">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>270</y>
            <width>41</width>
            <height>8</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 0, 0);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QGroupBox" name="gbIP">
          <property name="geometry">
           <rect>
            <x>2</x>
            <y>0</y>
            <width>165</width>
            <height>61</height>
           </rect>
          </property>
          <property name="title">
           <string>TCP/IP</string>
          </property>
          <widget class="QLabel" name="lbIP">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>21</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>IP:</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="leIP">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>30</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>192.168.100.100</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QRadioButton" name="rbTCPIP">
           <property name="geometry">
            <rect>
             <x>49</x>
             <y>0</y>
             <width>111</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bgSerial</string>
           </attribute>
          </widget>
         </widget>
         <widget class="QGroupBox" name="gbCOM">
          <property name="geometry">
           <rect>
            <x>2</x>
            <y>60</y>
            <width>165</width>
            <height>101</height>
           </rect>
          </property>
          <property name="title">
           <string>Serial</string>
          </property>
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>7</x>
             <y>30</y>
             <width>41</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Port : </string>
           </property>
          </widget>
          <widget class="QLabel" name="label_12">
           <property name="geometry">
            <rect>
             <x>4</x>
             <y>79</y>
             <width>161</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Param : 115200 8n1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QComboBox" name="cbPorts">
           <property name="geometry">
            <rect>
             <x>48</x>
             <y>29</y>
             <width>111</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="btRefreshPorts">
           <property name="geometry">
            <rect>
             <x>7</x>
             <y>52</y>
             <width>152</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Refresh ports</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QRadioButton" name="rbSerial">
           <property name="geometry">
            <rect>
             <x>49</x>
             <y>-2</y>
             <width>111</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bgSerial</string>
           </attribute>
          </widget>
         </widget>
         <widget class="QPushButton" name="btKV58FirmwareDownload">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>KV58 Firmware</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <widget class="Console" name="lvTexts">
         <property name="font">
          <font>
           <family>Fixedsys Excelsior 3.01-L2</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Console</class>
   <extends>QPlainTextEdit</extends>
   <header>console.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="bgSerial"/>
 </buttongroups>
</ui>
